node_modules/.yarn-integrity

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Can you remove this file from the PR for now? The `systemParams` below at least looks specific to Mac, so we wouldn't want to check that in.